### ISSUE TITLE
Missing confound file 

### ISSUE THREAD

**Issue by GerardYu** [nipreps/fmriprep#2675]
### What happened?

i was able to obtain the final *preproc_bold.nii.gz files. However i noticed that the confound files are missing in the func folder. Prior to running fMRIprep on the rsfMRI volumes, i've carried out field map corrections and removed the first 10 volumes

This did not happen with a previous version of fMRIprep.

### What command did you use?

```shell
fmriprep BIDS_folder output participant --skip_bids_validation --output-spaces MNI152NLin6Asym --participant-label $d --nthreads 8
```

### What version of fMRIPrep are you running?

21.0.0

### How are you running fMRIPrep?

Local installation ("bare-metal")

### Is your data BIDS valid?

Yes

### Are you reusing any previously computed results?

FreeSurfer

### Please copy and paste any relevant log output.

```shell
Errors
Node Name: fmriprep_wf.single_subject_010001_wf.func_preproc_task_rest_wf.bold_confounds_wf.dvars

File: /scratch/junhong.yu/LEMON2/BIDS/output/sub-010001/log/20220103-234101_481088ac-1a83-4321-8619-9ac83720967c/crash-20220103-235944-junhong.yu-dvars-1cbd6d5c-f30f-4281-a3a5-6da7409f1313.txt
Working Directory: /scratch/junhong.yu/LEMON2/BIDS/work/fmriprep_wf/single_subject_010001_wf/func_preproc_task_rest_wf/bold_confounds_wf/dvars
Inputs:

    figdpi: 100
    figformat: png
    figsize: (11.7, 2.3)
    in_file:
    in_mask:
    intensity_normalization: 1000.0
    remove_zerovariance: True
    save_all: False
    save_nstd: True
    save_plot: False
    save_std: True
    save_vxstd: False
    series_tr:

Traceback (most recent call last):
  File "/home/junhong.yu/fmriprepenv/lib/python3.7/site-packages/nipype/pipeline/plugins/multiproc.py", line 67, in run_node
    result["result"] = node.run(updatehash=updatehash)
  File "/home/junhong.yu/fmriprepenv/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 521, in run
    result = self._run_interface(execute=True)
  File "/home/junhong.yu/fmriprepenv/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 639, in _run_interface
    return self._run_command(execute)
  File "/home/junhong.yu/fmriprepenv/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 751, in _run_command
    f"Exception raised while executing Node {self.name}.\n\n{result.runtime.traceback}"
nipype.pipeline.engine.nodes.NodeExecutionError: Exception raised while executing Node dvars.

Traceback (most recent call last):
  File "/home/junhong.yu/fmriprepenv/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 398, in run
    runtime = self._run_interface(runtime)
  File "/home/junhong.yu/fmriprepenv/lib/python3.7/site-packages/nipype/algorithms/confounds.py", line 170, in _run_interface
    intensity_normalization=self.inputs.intensity_normalization,
  File "/home/junhong.yu/fmriprepenv/lib/python3.7/site-packages/nipype/algorithms/confounds.py", line 1030, in compute_dvars
    from nitime.algorithms import AR_est_YW
ModuleNotFoundError: No module named 'nitime'
```

**Comment by effigies** [comment]
It looks like we need to add nitime to the installation requirements. In the meantime, you can `pip install` it in your environment.

**Comment by effigies** [comment]
Closed by #2678.

Will be included in 21.0.1, expected next week.
